lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -1502,6 +1502,20 @@ void AtenSortIntOp::getCanonicalizationPatterns(RewritePatternSet &patterns,
   });
 }
 
+//===----------------------------------------------------------------------===//
+// AtenUnbindIntOp
+//===----------------------------------------------------------------------===//
+
+void AtenUnbindIntOp::getCanonicalizationPatterns(RewritePatternSet &patterns,
+                                                  MLIRContext *context) {
+  patterns.add(+[](AtenUnbindIntOp op, PatternRewriter &rewriter) {
```

Comment:
Same comment as in the other PR: https://github.com/llvm/torch-mlir/pull/2102#discussion_r1194437856